OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -0,0 +1,169 @@
+
+    # getent passwd 1000
+
+If that returns a username (which is the first field), use that user for the `useradd` command below. If not, create the `oracle` user manually. 
+
+    # useradd -u 1000 -g 1000 oracle 
+
+Run the following commands as root:
+
+    # mkdir -p /scratch/DockerVolume/SOAVolume/
+    # chown 1000:1000 /scratch/DockerVolume/SOAVolume/
+    # chmod 700 /scratch/DockerVolume/SOAVolume/
+
+# Database
+
+You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleDatabase/README.md) under docker/OracleDatabase for details on how to build Oracle Database image.
+
+The DB image created through above step need to be retagged from `container-registry.oracle.com/oracle/database:12.2.0.1-ee`  to `oracle/database:12.2.0.1-ee` before continuing with next steps.
+
+$ docker tag container-registry.oracle.com/oracle/database:12.2.0.1-e  oracle/database:12.2.0.1-ee
+
+## Running Oracle Database in a Docker container
+
+Create an environment file db.env.list
+
+    ORACLE_SID=<DB SID>
+    ORACLE_PDB=<PDB ID>
+    ORACLE_PWD=<password>
+    
+Sample Data will look like this:
+
+    ORACLE_SID=soadb
+    ORACLE_PDB=soapdb
+    ORACLE_PWD=Welcome1
+
+Sample Command to Start the Database is as follows
+
+     $ docker run --name soadb  --network=SOANet -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+     
+The above command starts a DB container attaching to a network and mounting a host directory as `/opt/oracle/oradata` for persistence. 
+It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+
+## SOA 12.2.1.3.0 Docker image
+
+You can either build the SOA image with the Dockerfile provided or use the already available Oracle SOA Suite (12.2.1.3.0) image in the [Oracle Container Registry](https://container-registry.oracle.com).
+
+## Creating a container for AdminServer
+
+Start a container to launch the Admin Server from the image created using above steps. The environment variables used to configure the domain are defined in `adminserver.env.list` file. Replace in `adminserver.env.list` the values for the Database and WebLogic passwords. 
+
+Create an environment file adminserver.env.list
+
+    CONNECTION_STRING=<Database Container Name>:<port#>/<ORACLE_PDB>
+    RCUPREFIX=<RCU_Prefix>
+    DB_PASSWORD=<database_sys_password>
+    DB_SCHEMA_PASSWORD=<soa-infra schema password>
+    ADMIN_PASSWORD=<admin_password>
+    MANAGED_SERVER=<managed servername>
+    DOMAIN_TYPE=<soa/osb/soaosb/soaess/soaessosb/bpm>
```

Comment:
You should explain what the different domain types are and how a customer would know which one to specify.